### DISCUSSION TITLE
Manage the working days and schedules

### DISCUSSION THREAD

**Discussion by skillforce** [nhn/tui.calendar#1227] [Q&A] [answered]
Hi guys, do we have any opportunity to set(and highlight) available hours in week view ( user sees all hours, but available- hightailed and only in available he can make schedule element)???

**Comment by adhrinae** [comment, marked as answer]
We offer [the `week.hourStart` and the `week.hourEnd` options.](https://github.com/nhn/tui.calendar/blob/main/docs/en/apis/options.md#weekhourstart) so maybe you can leverage them.

but there's no way to highlight certain areas in the calendar yet. perhaps you might need the 'background events' feature [like this](https://fullcalendar.io/docs/background-events)?

**Reply by skillforce** [reply]
@adhrinae thanks for your answer, I resolved it by using "createEvents" feature with customized css (extand to full width)
